.changelog/21816.txt

```diff
@@ -0,0 +1,9 @@
+```release-note:security
+mesh: Add `http.incoming.requestNormalization` to Mesh configuration entry to support inbound service traffic request normalization.
+```
+```release-note:security
+mesh: Add `contains` and `ignoreCase` to L7 Intentions HTTP header matching criteria to support configuration resilient to variable casing and multiple values.
```

Comment:
`CVE-2024-10006` has been reserved for this, we could add that it resolves it in the changelog.